compactor/src/query.rs

```diff
@@ -89,6 +94,44 @@ impl QueryableParquetChunk {
     pub fn max_time(&self) -> i64 {
         self.iox_metadata.time_of_last_write.timestamp_nanos()
     }
+
+    /// Convert to a QueryableParquetChunk
+    pub fn from_parquet_file_with_tombstones(
+        object_store: Arc<DynObjectStore>,
+        table_name: String,
+        parquet_file_with_tobstones: &ParquetFileWithTombstone,
```

Comment:
Nit: typo in parameter name through here:

```suggestion
        parquet_file_with_tombstones: &ParquetFileWithTombstone,
```